apps/foundation-demo/src/app/core/sidebar/containers/sidebar/sidebar.component.html

```diff
@@ -1,7 +1,9 @@
 <daff-sidebar class="sidebar" (escapePressed)="onClose()">
   <span class="sidebar__close" (click)="onClose()"><i class="fa fa-times"></i></span>
-  <a daff-sidebar-item (click)="onClose()" routerLink="/">Women's</a>
-  <a daff-sidebar-item (click)="onClose()" routerLink="/">Men's</a>
-  <a daff-sidebar-item (click)="onClose()" routerLink="/">Accessories</a>
-  <a daff-sidebar-item (click)="onClose()" routerLink="/">Sale</a>
+  <ul class="sidebar__list">
+    <li><a daff-sidebar-item (click)="onClose()" routerLink="/">Women's</a></li>
+    <li><a daff-sidebar-item (click)="onClose()" routerLink="/">Men's</a></li>
+    <li><a daff-sidebar-item (click)="onClose()" routerLink="/">Accessories</a></li>
+    <li><a daff-sidebar-item (click)="onClose()" routerLink="/">Sale</a></li>
```

Comment:
I think this should use the `daff-list` component instead, which is part of the footer pull request.